plugins/Jobs.cpp

```diff
@@ -770,7 +762,6 @@ void BedrockJobsCommand::process(SQLite& db) {
 
             // Add this object to our output
             STable job;
-            SINFO("Returning jobID " << result[c][0] << " from " << requestVerb);
```

Comment:
I am almost sure I use this log line and I don't think it is logged in other places, especially in non BWM flows. Re-add it?

Reply:
Are you sure it's necessary? PHP logs it several times, eg: https://www.expensify.com/_devportal/tools/logSearch/#sort=asc&size=10000&query=blob%3A%222415402389014186847%22%20AND%20timestamp%3A%5B2025-05-27T00%3A00%20TO%202025-05-28T23%3A59%5D

Reply:
I am not sure, that's why I said almost. That link is from BWM, which does log it, I was talking about non BWM flows. Checking now to see if we log it in those other flows....
Example https://www.expensify.com/_devportal/tools/logSearch/#query=request_id:(%2294613af849a98e57-SJC%22)+AND+timestamp:[2025-05-26T23:00:09.226Z+TO+2025-05-27T01:00:09.226Z]&index=logs_expensify-033946
We log it:
```
[ExpensiWorks] Expensify\Expensiworks\QueueManager - Got job ~~ created: '2025-05-27 00:00:05' data: '[_commitCounts: '[webrock: '63703487596']']' jobID: '5640302177949008878' lastRun: '' name: 'Expensiworks/SmartScanAmountAndCurrency' nextRun: '2025-05-27 00:00:05' priority: '1000' repeat: '' retryAfter: '+600 SECONDS'	
```

So this is good 👍